### ISSUE TITLE
runnyeye map doesnt load

### ISSUE THREAD

**Issue by trsBasicLine10** [nomns/nparse#14]
cant find it in the drop down list but map file is in the folder

**Comment by nomns** [comment]
Runnyeye should be called Liberated city of ...   That should be the name of it in game.  The program takes the "you have zoned into ..." and then uses the map_keys.ini file found in data/maps to choose a map file based on that text.

Is the Runnyeye map not loading when you zone into Runnyeye?

**Comment by trsBasicLine10** [comment]
correct the runnyeye map does not load. I tried restarting nparse and eq
client.
I am playing on project 1999. When i enter runnyeye I get  "Tou hsbe
entered Runnyeye Citadel"

Ok you told me that the names were mapped in that ini file and I set the
name to whet we get here on Project1999 and all is right with the world.

Sorry to trouble with a silly issue and thank you for responding so quickly.

Sincerely Fred Dixon.
